content/docs/proxy/google_serp_proxy/examples.md

```diff
@@ -142,56 +145,44 @@ Select this option by setting the `username` parameter to `groups-GOOGLE_SERP`.
 
 ```marked-tabs
 <marked-tab header="Node.js (axios)" lang="javascript">
-const HttpsProxyAgent = require("https-proxy-agent");
-const axios = require("axios");
+import axios from 'axios';
 
-const httpsAgent = new HttpsProxyAgent({
-    host: "proxy.apify.com",
-    port: "8000",
+const proxy = {
+    protocol: 'http',
+    host: 'proxy.apify.com',
+    port: 8000,
     // Replace <YOUR_PROXY_PASSWORD> below with your password
     // found at https://console.apify.com/proxy
-    auth: "groups-GOOGLE_SERP:<YOUR_PROXY_PASSWORD>"
-});
-
-const axiosWithProxy = axios.create({ httpsAgent });
-
-async function useProxy() {
-    const response = await axiosWithProxy.get(
-      `http://www.google.com/search`,
-      {
-        params: {
-          query: 'wikipedia',
-        },
-      }
-    );
-    console.log(response.data)
+    auth: { username: 'groups-GOOGLE_SERP', password: <YOUR_PROXY_PASSWORD> },
 };
 
-useProxy();
+const url = 'http://www.google.com/search';
+const params = { q: 'wikipedia' };
+
+const { data } = await axios.get(url, { proxy, params });
+
+console.log(data);
 </marked-tab>
 
 
 <marked-tab header="Node.js (got)" lang="javascript">
-const got = require("got");
-const HttpsProxyAgent = require("https-proxy-agent");
-
-// Replace <YOUR_PROXY_PASSWORD> below with your password
-// found at https://console.apify.com/proxy
-const proxyUrl = "http://groups-GOOGLE_SERP:<YOUR_PROXY_PASSWORD>@proxy.apify.com:8000"
-
-async function useProxy() {
-    const response = await got(`http://www.google.com/search`, {
-        searchParams: {
-          query: 'wikipedia',
-        },
-        agent: {
-            https: new HttpsProxyAgent(proxyUrl),
-        }
-    });
-    console.log(response.body);
+import got from 'got';
+import { HttpsProxyAgent } from 'hpagent';
+
+const agent = {
+    https: new HttpsProxyAgent({
+        // Replace <YOUR_PROXY_PASSWORD> below with your password
+        // found at https://console.apify.com/proxy
+        proxy: 'http://groups-GOOGLE_SERP:<YOUR_PROXY_PASSWORD>@proxy.apify.com:8000',
+    }),
 };
 
-useProxy();
+const url = 'http://www.google.com/search';
+const searchParams = { q: 'wikipedia' };
+
+const { body } = await got(url, { agent, searchParams });
+
+console.log(body);
```

Comment:
I would replace this with `got-scraping` so that we don't have to worry about got not working for now.

Reply:
Well - there's already `got-scraping` snippet came instead of `requestAsBrowser`, but we could ditch the `got` completely - `axios` is working fine, so it's `axios` for node, python, php?

Reply:
Yeah, we don't need to showcase multiple Node http clients.